layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/0001-ti_armv7_common.h-Don-t-hardcode-dev-0-but-use-mmcde.patch

```diff
@@ -1,29 +0,0 @@
-From 9eecb4e109f62e2e3f4094f4597afb7b819bf231 Mon Sep 17 00:00:00 2001
-From: Andrei Gherzan <andrei@resin.io>
-Date: Fri, 4 Mar 2016 15:33:46 +0100
-Subject: [PATCH 1/2] ti_armv7_common.h: Don't hardcode dev 0 but use mmcdev
-
-Signed-off-by: Andrei Gherzan <andrei@resin.io>
-
-Upstream-Status: Pending
-
----
- include/configs/ti_armv7_common.h | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
-diff --git a/include/configs/ti_armv7_common.h b/include/configs/ti_armv7_common.h
-index 6011cf8..2ca7762 100644
---- a/include/configs/ti_armv7_common.h
-+++ b/include/configs/ti_armv7_common.h
-@@ -61,7 +61,7 @@
- #define DEFAULT_MMC_TI_ARGS \
- 	"mmcdev=0\0" \
- 	"mmcrootfstype=ext4 rootwait\0" \
--	"finduuid=part uuid mmc 0:2 uuid\0" \
-+	"finduuid=part uuid mmc ${mmcdev}:2 uuid\0" \
```

Comment:
I think it is not a decision on our part if we keep or remove this patch when we talk about updating to the latest BSP. The change has been done at the BSP level (if this breaks resinHUP or not is independent of this PR) so if something is off with regards to the uEnv.txt mechanism we use at the moment in resinHUP then it seems like we need to adapt the mechanism to the existing situation rather than modify the BSP back to match the current resinHUP update mechanism.